core/src/test/scala/tech/beshu/ror/integration/CurrentUserMetadataAccessControlTests.scala

```diff
@@ -173,13 +173,15 @@ class CurrentUserMetadataAccessControlTests
       |    groups_endpoint: "http://${wiremock.getWireMockHost}:${wiremock.getWireMockPort}/groups"
       |    auth_token_name: "user"
       |    auth_token_passed_as: QUERY_PARAM
-      |    response_groups_json_path: "$$..groups[?(@.name)].name"
+      |    response_group_ids_json_path: "$$..groups[?(@.id)].id"
```

Comment:
IMO we should not modify the previous example. Still the case where group ID == group name is the default. 

We want to add additional tests obviously:
1. to check if our new solution works
2. to check if our solution is backward compatible 

Maybe this test suite is not the best place to do it.

Reply:
I added the 3rd Wiremock definition to test the new solution